wiki/releasing-versions.md

```diff
@@ -14,62 +16,10 @@ After the version is bumped, the release script automatically updates `CHANGELOG
 
 The command will prompt you for your git credentials. If you are using 2FA for git (which you should be) then your git password must be a [one time token](https://github.com/settings/tokens).
 
-That's it. The latest changes were published to GitHub, a new `git` tag now exists on GitHub, the new release can be installed from `npm`, and the [documentation site][docs] will update momentarily<sup>\*</sup>.
-
-<sup>_\* GitHub Pages sites are cached aggressively and can sometimes take a couple of minutes to update._</sup>
+That's it. The latest changes were published to GitHub, a new `git` tag now exists on GitHub, the new release can be installed from `npm`, and the [documentation site][docs] will update momentarily.
 
 ## `@elastic/eslint-plugin-eui`
 
 For information on releasing the eslint plugin checkout the readme in [packages/eslint-plugin/README.md](../packages/eslint-plugin/README.md)
```

Comment:
Should this be removed?

Reply:
yeah, it should - even if we still consume it, I don't think we have any current plans to fork or release. @AMoo-Miki is this something you already have a plan for?